libraries/botbuilder-ai/tests/luisRecognizer.test.js

```diff
@@ -490,119 +490,90 @@ describe('LuisRecognizer', function () {
         returnStatusCode(400);
 
         const context = new TestContext({ text: 'Hello world!' });
-        try {
-            await recognizer.recognize(context);
-            assert.fail('should have thrown');
-        } catch (error) {
-            const expectedError = `Response 400: The request's body or parameters are incorrect, meaning they are missing, malformed, or too large.`;
-            assert(error.message === expectedError, `unexpected error message thrown.`);
-        }
+        await assert.rejects(
+            recognizer.recognize(context),
+            Error(
```

Comment:
How is the diff when this doesn't match? I've noticed using `{message: "Expected message here"}` produces really useful diffs, just want to make sure that's the case with this as well. also, should really be `new Error`

Reply:
It's good!

![image](https://user-images.githubusercontent.com/40401643/117217416-32a10a80-adb6-11eb-9069-6af8a9c7d695.png)

> also, should really be new Error

I actually looked this one up, first because I wasn't completely sure. Per [the spec](https://tc39.es/ecma262/#sec-error-constructor) they're the same. However, as far as Bot Framework "standard practices" go, I'll go ahead and change it, if you prefer.

Reply:
Nice! Useful diff is all I desire, neither syntax is decidedly better IMO. So this is good!